src/v/utils/base64.h

```diff
@@ -31,3 +38,10 @@ ss::sstring base64_to_string(std::string_view);
 
 // base64 <-> iobuf
 ss::sstring iobuf_to_base64(const iobuf&);
+
+/// \brief Used to decode URL encoded base64 values
+///
+/// URL encoded base64 values use '-' and '_' instead of
+/// '+' and '/', respectively
+/// \throws base64_url_decoder_exception if
```

Comment:
`if ...` seems to be missing the rest of the sentence

Reply:
`if ... it feels like it`